configs/testing-library-compass/src/index.tsx

```diff
@@ -283,7 +283,9 @@ function createWrapper(
   const wrapperState = {
     globalAppRegistry: new AppRegistry(),
     localAppRegistry: new AppRegistry(),
-    preferences: new InMemoryPreferencesAccess(options.preferences),
+    preferences: new InMemoryPreferencesAccess(
+      options.preferences
+    ) as PreferencesAccess,
```

Comment:
For all those TS2742 cases, sometimes it was easier (and better) to make sure that internal interfaces are not leaking